setup.py

```diff
@@ -88,6 +90,23 @@ def data_folder(logger: logging.Logger) -> None:
         logger.warning("##Amend toml to download sample##")
         logger.warning("##Use shell script to download heavy##")
 
+    try:
+        logger.info("Building stops-geography lookup table")
+        # TODO: naming/access of function to be improved
+        create()
+    except Exception as e:
+        logger.warning(f"Build error: {e}")
+        pass
+
+    try:
+        schedule_builder = Schedule_Builder()
+        logger.info("Preparing punctuality data by stop")
+        # TODO: naming/access of method to be improved
+        schedule_builder.run()
```

Comment:
Left over code from previous commit. Doesn't stop setup.py running because its in the try statement. But not needed right now.

Returns:
Build error: 'Schedule_builder' object has no attribute 'run'.
..in the log file created.